examples/discovery-search-app/cypress/integration/pagination/spec.ts

```diff
@@ -70,6 +61,7 @@ describe('Pagination', () => {
 
       it('correctly requests the next page', () => {
         cy.get('@nextPageQueryObject')
+          // @ts-ignore
```

Comment:
Cypress's typing for `its()` only allows a limited selection of strings to be passed in, even though any valid property string will work. It looks like a proper fix might involve defining our own typing for `its`, but I didn't research too far down that rabbit hole.

Reply:
i'd be interested to investigate this more - `// @ts-ignore` will always throw a red flag for me (does this only apply to the following line?)

that being said - maybe if we leave a TODO comment or something similar to revisit this ill be ok with it for now. right now future developers might not know why this is here

Reply:
also where is the list of limited strings that can be passed in? if it is just `requestBody` maybe we might be using this in a way that cypress doesn't commit to supporting in future versions

Reply:
Found the type definition for `its()`, but I haven't tracked down a full list of strings yet.

Reply:
yeah - it looks like the following properties are the only ones that would be permitted by that definition https://github.com/cypress-io/cypress/blob/develop/cli/types/index.d.ts#L4467
```
  interface WaitXHR {
    duration: number
    id: string
    method: HttpMethod
    request: {
      body: string | ObjectLike
      headers: ObjectLike
    }
    requestBody: WaitXHR['request']['body']
    requestHeaders: WaitXHR['request']['headers']
    response: {
      body: string | ObjectLike
      headers: ObjectLike
    }
    responseBody: WaitXHR['response']['body']
    responseHeaders: WaitXHR['response']['headers']
    status: number
    statusMessage: string
    url: string
    xhr: XMLHttpRequest
  }
```

so i imagine drilling down into anything under `responseBody` wouldn't be allowed

Reply:
I'm doing some more digging on this right now. Looks like this problem is only exacerbated when the `wait`ed request gets aliased and fetched later. `cy.get('@alias')` has a return type of `JQuery<HTMLElement>`, which doesn't even have the `requestBody` parameter on it.

Here's an article with a possible workaround, but I still don't think it would let us drill father down than `requestBody`: https://medium.com/@NicholasBoll/cypress-io-making-aliases-type-safe-b6f5db07d580

Reply:
Here's another potential solution, but it would require us to define Interfaces for the requests that we want to send. Personally, I like the previous solution better, but I wanted to record it as an option:
https://github.com/cypress-io/cypress/issues/3068